doc/ctrlspace.txt

```diff
@@ -1443,6 +1443,12 @@ pure Vim Script (like `CtrlP`) thus it affects the performance too.
 
 General Assumptions~
 
+The file engine binaries have been compiled for various OS's and cpu
+types.  The programs available in the primary repository include Linux,
```

Comment:
... CPU types, but only those for Linux, MacOS and Windows under 64 bit architectures are available in the git repository. The versions for their 32 bit architecture counterparts, as well as for the FreeBSD, NetBSD and OpenBSD under 32 and 64 bit architectures can still be dowloaded ...